frontend/src/containers/Dashboard/Dashboard.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
```

Comment:
`import React, { Component } from 'react';` please. It's bugged me for a while and I did it the same way you did, but I think it's nicer if we do it this way?  Open to suggestions ?

Reply:
oops, sorry, fixed.